File: python/menor_de_tres.py
x: int
y: int
z: int

x = int(input("Primeiro valor: "))
y = int(input("Segundo valor: "))
z = int(input("Terceiro valor: "))

if (x < y) and (x < z):
    print(f"MENOR = {x}")
elif (y < x) and (y < z):
    print(f"MENOR = {y}")
else:
    print(f"MENOR = {z}")

File: java/menor_de_tres.java
import java.util.Locale;
import java.util.Scanner;

public class main {

	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Primeiro valor: ");
		int x = sc.nextInt();
		
		System.out.print("Segundo valor: ");
		int y = sc.nextInt();
		
		System.out.print("Terceiro valor: ");
		int z = sc.nextInt();
		
		
		if (x < y && x < z) {
			System.out.println("MENOR = " + x);
		} else if (y < z && y < z) {
			System.out.println("MENOR = " + y);
		} else {
			System.out.println("MENOR = " + z);
		}
	}

}
